File: password/nodejs/dns_lookup_pass.js
var http = require('https');
var querystring = require('querystring');

var domain = 'example.com';
var username = 'Your dns lookup api username';
var password = 'Your dns lookup api password';
var checkType = '_all';

var url = 'https://www.whoisxmlapi.com/whoisserver/DNSService';

var params = {
    type: checkType,
    domainName: domain,
    username: username,
    password: password
};

url = url + '?' + querystring.stringify(params);

http.get(url, function(response) {
    var str = '';
    response.on('data', function(chunk) {
        str += chunk;
    });
    response.on('end', function() {
        console.log(str);
    });
}).end();

File: password/python/dns_lookup_pass.py
try:
    from urllib.request import urlopen, pathname2url
except ImportError:
    from urllib import pathname2url
    from urllib2 import urlopen

domain = 'example.com'
username = 'Your dns lookup api username'
password = 'Your dns lookup api password'
checkType = '_all'

url = 'http://www.whoisxmlapi.com/whoisserver/DNSService?'\
    + 'type=' + pathname2url(checkType)\
    + '&domainName=' + pathname2url(domain)\
    + '&username=' + pathname2url(username)\
    + '&password=' + pathname2url(password)

print(urlopen(url).read().decode('utf8'))
